python/nammu/controller/NammuController.py

```diff
@@ -361,9 +360,8 @@ def saveAsFile(self, event=None):
                             JOptionPane.YES_NO_OPTION)
                 if reply == JOptionPane.NO_OPTION:
                     return
-            filename = self.force_atf_extension(filename)
-            self.currentFilename = filename
-            self.view.setTitle(basename)
```

Comment:
I'm guessing this isn't final, but now you no longer need `basename` (defined in line 352).